.gitignore

```diff
@@ -24,3 +24,4 @@ geckodriver.log
 node_modules/
 packages/.artifacts
 packages/*/build.log
+dist/
```

Comment:
Not entirely related to this PR:

I think we should also switch to building the main pyodide files into a folder named `dist` rather than building into `build`. I found this a bit confusing -- in particular I think newcomers may not find it intuitive what the difference between `pyodide-build` and `build` is, whereas `dist` is clearer.

Reply:
> I think we should also switch to building the main pyodide files into a folder named dist rather than building into build

Yeah, definitely.